File: inflearn/javascript/1-04/main.js
const ps = require('../modules/ps');

const solution = (n, arr) => {
  let answer = '';
  arr.forEach((element) => {
    // 문자열 뒤집기 (문자열을 배열로 바꿔서 뒤집은뒤 다시 문자열로)
    answer += element.split('').reverse().join('') + '\n';
  });
  return answer;
};

ps.main(async () => {
  let n = parseInt(await ps.readLine()); // 문자열을 입력 받아 정수 반환
  let arr = [];
  for (let i = 0; i < n; i++) {
    let input = await ps.readLine(); // 문자열 입력
    arr.push(input);
  }

  console.log(solution(n, arr));
});

File: inflearn/java/1-04/Main.java
import java.util.*;

public class Main {
    public static String solution(String str) {
        String answer = "";
        int len = str.length() - 1;
        for (int i = len; i >= 0; i--)
            answer += str.charAt(i);
        return answer;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            String input = sc.next();
            System.out.println(solution(input));
        }
        sc.close();
    }
}
